url/urltestdata.txt

```diff
@@ -1,402 +0,0 @@
-# FORMAT NOT DOCUMENTED YET (parser is urltestparser.js)
-
-# Based on http://trac.webkit.org/browser/trunk/LayoutTests/fast/url/script-tests/segments.js
```

Comment:
For now I've restored the comments, except for the ones that were obviously wrong (saying "this should fail" when due to previous updates they no longer fail). Some of them are reasonably valuable, talking about why something fails.